qubit-sdk/src/main/java/com/qubit/android/sdk/api/tracker/EventTracker.kt

```diff
@@ -31,6 +37,23 @@ interface EventTracker {
       ignoreSegments: Boolean? = null
   )
 
+  /**
+   * Fetch placement defined for given criteria.
+   * Method is asynchronous, result is returned through passed callback functions.
+   * @param placementId The unique ID of the placement.
+   * @param mode The mode to fetch placements content with. Defaults to [PlacementMode.LIVE]
+   * @param previewOptions Additional criteria used to query for the placement.
+   * @param onSuccess Callback invoked when query succeeds. Contains [Placement] object or null if no placement meets given criteria.
+   * @param onError Callback invoked when query fails.
+   */
+  fun getPlacement(
+      placementId: String,
+      mode: PlacementMode?,
```

Comment:
PlacementMode, PlacementPreviewOptions, Placement are element of public API of the SDK, while they are declared in package `internal`.
There was such convention here, that public API elements are in `api` package.

Reply:
Right. I did it how it has been done with experiences which was incorrect as well.

Moving Placement* classes and keeping Experience* classes as they are so backward compatibility is preserved.